### PR TITLE
BasePath modified by env value

### PR DESCRIPTION
**Pull request by pjainxido**

### CODE REVIEW

**Comment by blackcowmooo** [comment]
Argo sync fail

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "jest",
-    "dev": "NEXT_PUBLIC_ENV=dev next",
+    "dev": "NEXT_PUBLIC_ENV_ROOT_PATH=/pr/18 NEXT_PUBLIC_ENV=dev next",
```
Comment (micalgenus) [inline package.json]: ?

**Comment by blackcowmooo** [comment]
Argo sync fail

**Comment by blackcowmooo** [comment]
Argo sync fail

File: package.json (hunk 1 above)
Comment (micalgenus) [inline package.json]:
```suggestion
    "dev": "NEXT_PUBLIC_ENV=dev next",
```

**Review by micalgenus — APPROVED** [review]

**Comment by blackcowmooo** [comment]
Argo sync fail

**Pull request merged**